lib/internal/fs/cp/cp-sync.js

```diff
@@ -226,7 +229,7 @@ function mayCopyFile(srcStat, src, dest, opts) {
 }
 
 function copyFile(srcStat, src, dest, opts) {
-  copyFileSync(src, dest);
+  copyFileSync(src, dest, COPYFILE_FICLONE);
```

Comment:
This seems like something that users should pass in via `cp()`'s options argument, and not something that Node core sets unconditionally.

Reply:
Okay. I withdeaw this pr. Thank you for your review!